pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -34,7 +33,7 @@ var genericPreflightChecks = [...]Check{
 		configKeySuffix:  "check-bundle-extracted",
 		checkDescription: "Checking if CRC bundle is extracted in '$HOME/.crc'",
 		check:            checkBundleExtracted,
-		fixDescription:   "Extracting bundle from the CRC executable",
+		fixDescription:   "Extracting bundle from the cache or current directory",
```

Comment:
or just "Extracting CodeReady Containers bundle"?

Reply:
@cfergeau this is part of fix in setup so I think explicitly mention the bundle directory make more sense then just putting a general statement.

Reply:
thinking just now that "current directory" is not accurate. And when things "just work", I don't think the user cares a lot about where things are, they only care that this is being extracted. Could be a good debug message toe report if `checkBundleExtracted` when returning nil because the bundle could not be found actually "Not extracting bundle as we couldn't find it in ~/.crc/cache nor in the same directory as the crc executable"